OpenRA.Mods.Common/Widgets/Logic/Settings/GamePlaySettingsLogic.cs

```diff
@@ -0,0 +1,136 @@
+#region Copyright & License Information
+/*
+ * Copyright (c) The OpenRA Developers and Contributors
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using OpenRA.Widgets;
+
+namespace OpenRA.Mods.Common.Widgets.Logic
+{
+	public class GameplaySettingsLogic : ChromeLogic
+	{
+		[FluentReference]
+		const string AutoSaveIntervalOptions = "auto-save-interval.options";
+
+		[FluentReference]
+		const string AutoSaveIntervalDisabled = "auto-save-interval.disabled";
+
+		[FluentReference]
+		const string AutoSaveIntervalMinuteOptions = "auto-save-interval.minute-options";
+
+		[FluentReference]
+		const string AutoSaveFileNumberOptions = "auto-save-file-number.options";
+
+		[FluentReference]
+		const string AutoSaveFileNumberDisabled = "auto-save-file-number.disabled";
+		readonly int[] autoSaveSeconds = { 0, 10, 30, 45, 60, 120, 180, 300, 600 };
+
+		[ObjectCreator.UseCtor]
+		public GameplaySettingsLogic(Action<string, string, Func<Widget, Func<bool>>, Func<Widget, Action>> registerPanel, string panelID, string label)
+		{
+			registerPanel(panelID, label, InitPanel, ResetPanel);
+		}
+
+		Func<bool> InitPanel(Widget panel)
+		{
+			var scrollPanel = panel.Get<ScrollPanelWidget>("SETTINGS_SCROLLPANEL");
+			SettingsUtils.AdjustSettingsScrollPanelLayout(scrollPanel);
+
+			// Setup dropdown for auto-save interval
+			var autoSaveIntervalDropDown = panel.Get<DropDownButtonWidget>("AUTO_SAVE_INTERVAL_DROP_DOWN");
+
+			autoSaveIntervalDropDown.OnMouseDown = _ =>
+				ShowAutoSaveIntervalDropdown(autoSaveIntervalDropDown, autoSaveSeconds, scrollPanel);
+
+			autoSaveIntervalDropDown.GetText = () => GetMessageForAutoSaveInterval(Game.Settings.Gameplay.AutoSaveInterval);
+
+			// Setup dropdown for auto-save nr.
+			var autoSaveNoDropDown = panel.Get<DropDownButtonWidget>("AUTO_SAVE_FILE_NUMBER_DROP_DOWN");
+
+			autoSaveNoDropDown.OnMouseDown = _ =>
+				ShowAutoSaveFileNumberDropdown(autoSaveNoDropDown, Enumerable.Range(0, 21), scrollPanel);
+
+			autoSaveNoDropDown.GetText = () => GetMessageForAutoSaveFilesNo(Game.Settings.Gameplay.AutoSaveFileNumber);
+
+			autoSaveNoDropDown.IsDisabled = () => Game.Settings.Gameplay.AutoSaveInterval == 0;
+
+			return () => true;
```

Comment:
```suggestion
			return () => false;
```

we don't need to reload the game